src/services/schema/update/database.js

```diff
@@ -70,7 +70,10 @@ class Database {
       if (options.dbPassword) {
         connectionUrl += `:${options.dbPassword}`;
       }
-      connectionUrl += `@${options.dbHostname}`;
+      if (options.dbUser || options.dbPassword) {
+        connectionUrl += '@';
+      }
+      connectionUrl += options.dbHostname;
```

Comment:
Before changing this, we couldn't connect to a database without credentials using the test helper.
Now we are ISO with the documentation: https://www.mongodb.com/docs/manual/reference/connection-string/#standard-connection-string-format

Reply:
The helper was working for other tests since we never went through the connectionString computing on them.
Here if we pass the same URL `mongodb://localhost:27016` it was transformed to `mongodb://@localhost:27016` which the driver does not accept.